sdk/cosmosdb/arm-cosmosdb/package.json

```diff
@@ -3,16 +3,16 @@
   "sdk-type": "mgmt",
   "author": "Microsoft Corporation",
   "description": "A generated SDK for CosmosDBManagementClient.",
-  "version": "16.1.1",
+  "version": "17.0.0-beta.1",
   "engines": {
     "node": ">=18.0.0"
   },
   "dependencies": {
-    "@azure/abort-controller": "^1.0.0",
-    "@azure/core-auth": "^1.6.0",
-    "@azure/core-client": "^1.7.0",
     "@azure/core-lro": "^2.5.4",
+    "@azure/abort-controller": "^2.1.2",
```

Comment:
why we have so many dependencies reorder change? this was released not long before